### PR TITLE
Multicopter kits - new structure

### PR DESCRIPTION
**Pull request by hamishwillee**
This creates a "Kits" folder for MC kits and build logs "soemwhat like" the proposed new structure.

The somewhat like is that this includes a few non-kit builds. I think this is a good first step - at least until we have guidelines for a DIY build.

As part of this I have removed a lumenier build and a dji flamewheel build - because those frames aren't available and the flight controllers are discontinued. I'm tempted to remove more :-)

Note @sfuhrer there is more to do. But this is ready for a sanity check. Next step would be to add a peer "Off the shelf vehicles" 
(Arguably of course this could all go in assembly). Eventually remove the complete vehicle page.

Thoughts?

### CODE REVIEW

File: en/flight_controller/cuav_v5_nano.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ Main FMU Processor: STM32F765◦32 Bit Arm® Cortex®-M7, 216MHz, 2MB memory, 51
 
 ## Where to Buy
 
-<!-- [CUAV Store](https://store.cuav.net/index.php?id_product=95&id_product_attribute=0&rewrite=cuav-new-pixhack-v5-autopilot-m8n-gps-for-fpv-rc-drone-quadcopter-helicopter-flight-simulator-free-shipping-whole-sale&controller=product&id_lang=1) -->
+[CUAV Store](hhttps://store.cuav.net/shop/v5-nano/)
```
Comment (hamishwillee) [inline en/flight_controller/cuav_v5_nano.md:65]:
```suggestion
[CUAV Store](https://store.cuav.net/shop/v5-nano/)
```

File: en/SUMMARY.md [hunk 2]
```diff
@@ -57,17 +57,16 @@
...
-    * [X500 (Pixhawk 4)](frames_multicopter/holybro_x500_pixhawk4.md)
-    * [S500 V2 (Pixhawk 4)](frames_multicopter/holybro_s500_v2_pixhawk4.md)
-    * [DJI F450 (CUAV v5+)](frames_multicopter/dji_f450_cuav_5plus.md)
-    * [DJI F450 (CUAV v5 nano)](frames_multicopter/dji_f450_cuav_5nano.md)
-    * [QAV250 (Pixhawk4 Mini) - Discontinued](frames_multicopter/holybro_qav250_pixhawk4_mini.md)
-    * [DJI F450 + RTK \(Pixhawk 3 Pro\)](frames_multicopter/dji_flamewheel_450.md)
-    * [QAV250 \(Pixhawk Mini\)](frames_multicopter/lumenier_qav250_pixhawk_mini.md)
-    * [QAV-R 5" Racer (Pixracer)](frames_multicopter/qav_r_5_kiss_esc_racer.md)
-    * [Omnicopter](frames_multicopter/omnicopter.md)
+    * [Kits](../frames_multicopter/kits.md)
```
Comment (hamishwillee) [inline en/SUMMARY.md:60]:
```suggestion
    * [Kits](frames_multicopter/kits.md)
```

File: en/frames_plane/diy_builds.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+Do-it-yourself (DIY) builds describe how to assemble a vehicle from parts that are sourced separately.
+
+At the easier end of the scale of DIY builds are those that take an existing RC vehicle and graft on a flight controller.
+At the harder end of the scale are "unique" vehicles where some parts may be fabricated using 3D printers.
+Somewhere in the middle are builds that use separate but readily available components: frame, motors, ESC, propellers, flight controller, and so on.
+
+DIY Builds:
+
+- [Reptile Dragon 2 (ARK6X)](../frames_plane/reptile_dragon_2.md)
+- [Turbo Timber Evolution (../Pixhawk 4 Mini)](frames_plane/turbo_timber_evolution.md)
```
Comment (hamishwillee) [inline en/frames_plane/diy_builds.md:12]:
```suggestion
- [Turbo Timber Evolution (../Pixhawk 4 Mini)](../frames_plane/turbo_timber_evolution.md)

```

**Comment by hamishwillee** [comment]
@sfuhrer I think this is good to get in. Happy to take comments as a post process!

**Comment by sfuhrer** [comment]
Yes that's totally okay, thanks for addressing it.

**Pull request merged**